lua/neo-tree/sources/common/node_expander.lua

```diff
@@ -0,0 +1,85 @@
+local log = require("neo-tree.log")
```

Comment:
I extracted this nodes expander as something that is filesystem agnostic and could be used to load also other lazy sources.

Reply:
nice